deposit/hal/metadata.py

```diff
@@ -27,26 +27,9 @@
 
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/deposit/hal/metadata.py
+++ b/deposit/hal/metadata.py
@@ -25,11 +25,11 @@
 
 from __future__ import unicode_literals
 
+from deposit.hal.forms import HALForm
 from deposit.sword.metadata import addChild
 from deposit.sword.metadata import MetadataFormatter
 from lxml import etree
 from papers.models import Paper
-from deposit.hal.forms import HALForm
 
 ENS_HAL_ID = 59704
 
```